src/Symfony/Component/Serializer/Context/Normalizer/AbstractNormalizerContextBuilder.php

```diff
@@ -164,4 +164,13 @@ public function withIgnoredAttributes(?array $ignoredAttributes): static
     {
         return $this->with(AbstractNormalizer::IGNORED_ATTRIBUTES, $ignoredAttributes);
     }
+
+    /**
+     * Configures flag to prevent fields that are nullable to receive a null
+     * fallback value when no parameters are provided for it.
```

Comment:
Configures requiring all properties to be listed in the input instead of falling back to null for nullable ones.